### ISSUE TITLE
[BUG] Webhook error

### ISSUE THREAD

**Issue by claytondukes** [Kav-K/GPTDiscord#431]
**Describe the bug**
Every so often, I get a console error about invalid web hooks. From what I can tell, everything still works though.

I just can't tell where they are coming from.

I don't know if it coincides with some users typing something and the bot responds with `The API returned an invalid response: 400: Bad Request` in discord. But when I see those, maybe it's because the user is trying to generate something that violates Dalle policy? e.g.: a user typed `/dalle draw prompt: a big tit lev rag`. Another typed `/dalle draw prompt: synderella a demon hunter with a bow in a Chris outfit`

```
Task exception was never retrieved
future: <Task finished name='discord-ui-view-timeout-2c0b23a5073ec30e9bd93c7478d27770' coro=<SaveView.on_timeout() done, defined at /usr/local/lib/python3.10/site-packages/services/image_service.py:265> exception=HTTPException('401 Unauthorized (error code: 50027): Invalid Webhook Token')>
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/services/image_service.py", line 282, in on_timeout
    await self.ctx.edit(view=new_view)
  File "/usr/local/lib/python3.10/site-packages/discord/interactions.py", line 428, in edit_original_response
    data = await adapter.edit_original_interaction_response(
  File "/usr/local/lib/python3.10/site-packages/discord/webhook/async_.py", line 221, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 401 Unauthorized (error code: 50027): Invalid Webhook Token
Task exception was never retrieved
future: <Task finished name='discord-ui-view-timeout-2ef85ac0a251a85227e5f27fb0877f9d' coro=<SaveView.on_timeout() done, defined at /usr/local/lib/python3.10/site-packages/services/image_service.py:265> exception=HTTPException('401 Unauthorized (error code: 50027): Invalid Webhook Token')>
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/services/image_service.py", line 282, in on_timeout
    await self.ctx.edit(view=new_view)
  File "/usr/local/lib/python3.10/site-packages/discord/interactions.py", line 428, in edit_original_response
    data = await adapter.edit_original_interaction_response(
  File "/usr/local/lib/python3.10/site-packages/discord/webhook/async_.py", line 221, in request
    raise HTTPException(response, data)
discord.errors.HTTPException: 401 Unauthorized (error code: 50027): Invalid Webhook Token
```

**Comment by Kav-K** [comment]
It looks like its coming from when views try to expire, but the view object has expired itself, something like that, will look into it after some of this OpenAI stuff rolls over

**Comment by Kav-K** [comment]
This should be fixed in release v12.3.5
